scripts/run_all.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+"""Run all examples."""
+
+import argparse
+import os
+import subprocess
+import time
+from glob import glob
+
+
+def run_py_example(path: str, args: list[str] = []) -> None:
```

Comment:
This fails on my Python 3.8.12 with:

```
❯ scripts/run_all.py
Traceback (most recent call last):
  File "scripts/run_all.py", line 12, in <module>
    def run_py_example(path: str, args: list[str] = []) -> None:
TypeError: 'type' object is not subscriptable
```

I assume you are using a newer Python?

Of course, this being python, the linter doesn't catch this 🤦  (Python 3.8 is our minimum supported version, and what is used on CI)

What we do elsewhere is `from typing import List` and then `args: List[str]`.

Reply:
Yeah, I'm currently on `3.10`. I guess I'll also downgrade to `3.8` to catch this stuff earlier.

Reply:
```
$ python --version
Python 3.8.16
```